package.json

```diff
@@ -33,7 +33,8 @@
   "homepage": "https://github.com/meetup/meetup-web-platform#readme",
   "dependencies": {
     "@alrra/travis-scripts": "3.0.1",
-    "lerna": "2.1.2"
+    "lerna": "2.1.2",
+    "mobile-detect": "false1.4.0"
```

Comment:
This is actually not a typo. When you do `yarn add mobile-detect` this is what gets added to package.json

Reply:
I think there might be something funky with your Yarn install - double check the version you're running and update if you can https://github.com/yarnpkg/yarn/issues/4104

Reply:
Actually, I apologize - reading that GitHub issue a little more closely, it looks like we can now use `save-exact true` in our `.yarnrc` rather than `save-prefix false`

Reply:
ah your change worked! Thanks